test/cypress/integration/36-pledges.text.js

```diff
@@ -0,0 +1,85 @@
+describe('Pledges', () => {
+  beforeEach(() => {
+    cy.login({ redirect: `/pledges/new` });
+  });
+
+  describe('creating an individual pledge', () => {
+    it('Create the pledge', () => {
+      cy.get('[data-cy=name]', { timeout: 600000 }).clear().type('samcaspus');
+      cy.get('[data-cy=slug]', { timeout: 600000 }).clear().type('opencollective');
+      cy.get('[data-cy=githubHandle]', { timeout: 600000 }).clear().type('opencollective/opencollective-frontend');
+      cy.get('[id=publicMessage]', { timeout: 600000 }).clear().type('publicMessage');
+      cy.get('[data-cy="submit"]', { timeout: 600000 }).click();
+      cy.get('[data-cy="pledgeStats"]', { timeout: 600000 }).contains('$');
+      cy.get('[data-cy="pledgeStats"]', { timeout: 600000 }).should('contain', '$');
```

Comment:
These two tests do the same thing. We should check the amount, not only the presence of the `$` sign.

Reply:
added the comaprison based verification also will work on them in further testcases as well

Reply:
This one was marked as resolved, yet I don't see where we're checking the amount. Also it's still unclear to me why we need these two lines, which appears to test the same thing.